drivers/volume/portworx/portworx.go

```diff
@@ -1287,7 +1290,7 @@ func (d *portworx) ValidateCreateVolume(volumeName string, params map[string]str
 	}
 
 	// Validate Device Path for a Volume
-	if vol.Spec.ProxySpec != nil && vol.Spec.ProxySpec.ProxyProtocol == api.ProxyProtocol_PROXY_PROTOCOL_PURE_BLOCK {
+	if vol.Spec.ProxySpec != nil && vol.Spec.ProxySpec.ProxyProtocol == api.ProxyProtocol_PROXY_PROTOCOL_PURE_BLOCK && !isRaw {
```

Comment:
Why we are not checking device path for raw block volume ? it will be a direct access volume right ..

Reply:
Done.